tools/lighter/lighter.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="lighter" name="lighter" version="1.0">
+    <description>can correct the reads containing sequence errors</description>
+    <command>
+       lighter
+       #for $file in $r:
+           -r $file
+       #end for
+       #if str($kmer.k_select) == "smallk":
+           -k $kmer.k_parameter_string

```

Comment:
Yes they do! Thanks for all your work today! :) Reading through my mails now ...